source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB specific information to users,
```

Comment:
[Question]

Do we need to say "to users"? If we omit, this is more minimal and I think becomes slightly clearer that we're talking to the reader.

Reply:
Omitted!